### PR TITLE
Add initial Wisdom For Sale .rsh and .mjs files

### PR DESCRIPTION
**Pull request by Craig-Elre**
This ran properly in Ubuntu and output the role of seller and the network as Algo as intended.

This is intended to be a test of the process of creating a branch off master in GitHub Desktop separated from my forked branch, but adds files required for Wisdom For Sale.

### CODE REVIEW

File: examples/docs-wisdom-1/index.mjs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const commonInteract = {};
+
+// Seller
+if (role === 'seller') {
+  const sellerInteract = { 
+    ...commonInteract };
+		
+  const acc = await stdlib.newTestAccount(stdlib.parseCurrency(1000));
+  const ctc = acc.contract(backend);
+  await backend.Seller(ctc, sellerInteract);
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.mjs:19]: You should use the new syntax... `await ctc.participants.Seller(sellerInteract)`

File: examples/docs-wisdom-1/index.mjs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const stdlib = loadStdlib(process.env);
+console.log(`The consensus network is ${stdlib.connector}.`);
+
+const commonInteract = {};
+
+// Seller
+if (role === 'seller') {
+  const sellerInteract = { 
+    ...commonInteract };
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.mjs:15]: I think standard JS formatting puts this `};` on the next line

**Review by jeapostrophe — CHANGES_REQUESTED** [review]
The CI will fail on this because there's no `index.txt` file. I will make a change on your branch to add it, because it is difficult to do if you haven't yet set up a dev build of Reach on your machine.

It would be best if you and @TheChronicMonster would name your branches DCT-XYZ where XYZ is the number of the Jira card for the task.

File: examples/docs-wisdom-1/index.mjs [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    ...commonInteract 
+  };
+		
+  const acc = await stdlib.newTestAccount(stdlib.parseCurrency(1000));
+  const ctc = acc.contract(backend);
+  await ctc.participants.Seller(sellerInteract)
+}
+	
+// Buyer
+else {
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.mjs:24]: Can you make this `} else {`?

File: examples/docs-wisdom-1/index.mjs [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  const buyerInteract = {
+	...commonInteract 
+  };
+
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.mjs:28]: Do you want it to be empty from here?

File: examples/docs-wisdom-1/index.mjs [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+		
+  const acc = await stdlib.newTestAccount(stdlib.parseCurrency(1000));
+  const ctc = acc.contract(backend);
+  await ctc.participants.Seller(sellerInteract)
+}
+	
+// Buyer
+else {
+  const buyerInteract = {
+	...commonInteract
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.mjs:26]: Bad indentation

File: examples/docs-wisdom-1/index.rsh [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+  ...commonInteract,
+};
+
+export const main = Reach.App(() => {
+  const S = Participant('Seller', sellerInteract);
+  const B = Participant('Buyer', buyerInteract);
+  init();
+
+  exit();
+});
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.rsh:17]: No newline at end of file

File: examples/docs-wisdom-1/index.mjs [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+  await ctc.participants.Seller(sellerInteract)
+}
+	
+// Buyer
+else {
+  const buyerInteract = {
+	...commonInteract 
+  };
+
+};
```
Comment (jeapostrophe) [inline examples/docs-wisdom-1/index.mjs:29]: Newline

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: examples/docs-wisdom-1/index.mjs (hunk 4 above)
Comment (Craig-Elre) [reply examples/docs-wisdom-1/index.mjs:28]: This is intentionally left blank at this point. It gets added to later in the tutorial.

**Review by jeapostrophe — APPROVED** [review]

**Pull request merged**